src/Azure.Functions.Cli/StaticResources/Dockerfile.dotnet9Isolated

```diff
@@ -0,0 +1,14 @@
+FROM mcr.microsoft.com/dotnet/sdk:9.0-preview AS installer-env
+
+COPY . /src/dotnet-function-app
+RUN cd /src/dotnet-function-app && \
+mkdir -p /home/site/wwwroot && \
+dotnet publish *.csproj --output /home/site/wwwroot
```

Comment:
I think it might be a good idea to include `-c release` in the `dotnet publish` command